Authentication.md

```diff
@@ -278,39 +278,12 @@ The IAM access token is added to each outbound request in the `Authorization` he
    Authorization: Bearer <IAM-access-token>
 ```
 
-### Compute Resource Token Retrieval Algorithm
```

Comment:
No need to go into any detail about how the cr token is retreived since it's just read from a file.